recipe/meta.yaml

```diff
@@ -77,12 +70,12 @@ test:
     # although not listed in the README among the backends. We add a quick test here.
     #
     # numpy import jax in source code which causes test to fail for win
-    - python -c "import os; os.environ['KERAS_BACKEND'] = 'numpy'; import keras;"  # [not win]
-    - python -c "import os; os.environ['KERAS_BACKEND'] = 'jax'; import keras;"    # [not win]
+    - python -c "import os; os.environ['KERAS_BACKEND'] = 'numpy'; import keras;"  # [not (win or s390x)]
```

Comment:
how about running the [integration tests](https://github.com/keras-team/keras/tree/master/integration_tests)? they don't look too heavy.